src/api/modelTypes.js

```diff
@@ -276,6 +276,8 @@ export type Narrow = $ReadOnlyArray<NarrowElement>;
  */
 export type ReactionType = 'unicode_emoji' | 'realm_emoji' | 'zulip_extra_emoji';
 
+export type EmojiType = 'unicode_emoji' | 'realm_emoji';
```

Comment:
Created new type because name `ReactionType` doesn't make sense when we have to render emoji for `EmojiAutocomplete`. I will prefer to rename `ReactionType ` to `EmojiType `. 

On the other hand when I hear `EmojiType `, first thing comes to my mind is either `unicode` or `image`. But while aggregating reactions we are just using `ReactionType ` as type, so made it `EmojiType = 'unicode_emoji' | 'realm_emoji';`. I thinks we should make this change at the time of aggregating reactions, i.e set type as either `unicode` or `image`. Thoughts?